dbt/dbt_project.yml

```diff
@@ -1,4 +1,4 @@
-name: 'athena'
+name: 'ccao_data_athena'
```

Comment:
Per the bug [that Jean filed](https://github.com/dbt-athena/dbt-athena/issues/607), `dbt-athena` can't have the _project_ named Athena or the name spacing gets messed up. Annoyingly the project name can't contain dashes, so prefixing it with `ccao_data_` for now.